### PR TITLE
Fixes ScrollBar template hard-coded size values

### PR DESCRIPTION
**Pull request by wieslawsoltes**
- What does the pull request do?
Fixes ScrollBar template hard-coded size values.
- What is the current behavior?
The width and height of ScrollBar template are hard to change.
- What is the updated/expected behavior with this PR?
User can easily change width and height of ScrollBar.
- How was the solution implemented (if it's not obvious)?
Added theme resources for ScrollBar width and height.

### CODE REVIEW

File: tests/Avalonia.Layout.UnitTests/FullLayoutTests.cs [hunk 1]
```diff
@@ -113,17 +113,17 @@ public void Test_ScrollViewer_With_TextBlock()
                 Assert.Single(presenters);
 
                 var presenter = presenters[0];
-                Assert.Equal(new Size(190, 190), presenter.Bounds.Size);
+                Assert.Equal(new Size(196, 196), presenter.Bounds.Size);
```
Comment (jmacato) [inline tests/Avalonia.Layout.UnitTests/FullLayoutTests.cs]: Why was this `190` instead of the `196`?

Comment (wieslawsoltes) [reply tests/Avalonia.Layout.UnitTests/FullLayoutTests.cs]: I have changed grid column and row size from fixed to auto in xaml template.

Comment (wieslawsoltes) [reply tests/Avalonia.Layout.UnitTests/FullLayoutTests.cs]: That is the result of adding `ScrollBarWidth` and `ScrollBarHeight` to theme accent, the theme accent is not applied int the test so size is different. You can add to the test `Test_ScrollViewer_With_TextBlock`:
```C#
                window.Resources["ScrollBarWidth"] = 10.0;
                window.Resources["ScrollBarHeight"] = 10.0;

                window.Show();
                window.LayoutManager.ExecuteInitialLayoutPass(window);
```

**Review by grokys — COMMENTED** [review]
Do we need a separate width and height? Is there a case where we'd want horizontal scrollbars to be a different height to the vertical scrollbar width?

File: src/Avalonia.Themes.Default/Accents/BaseDark.xaml [hunk 2]
```diff
@@ -52,5 +52,8 @@
         <sys:Double x:Key="FontSizeSmall">10</sys:Double>
         <sys:Double x:Key="FontSizeNormal">12</sys:Double>
         <sys:Double x:Key="FontSizeLarge">16</sys:Double>
+
+        <sys:Double x:Key="ScrollBarWidth">10</sys:Double>
```
Comment (grokys) [inline src/Avalonia.Themes.Default/Accents/BaseDark.xaml]: Do we need a separate width and height? Is there a case where we'd want horizontal scrollbars to be a different height to the vertical scrollbar width?

Comment (wieslawsoltes) [reply src/Avalonia.Themes.Default/Accents/BaseDark.xaml]: Will test both cases and see what works best.

**Comment by wieslawsoltes** [comment]
@grokys I changed to `ScrollBarThickness` instead of two separate values `ScrollBarWidth` and `ScrollBarHeight`.

**Review by grokys — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**